examples/live-cursors.ts

```diff
@@ -0,0 +1,27 @@
+import Spaces from '@ably-labs/spaces';
+import { Realtime } from 'ably';
+
+import renderCursor from './my-application';
+
+// Create Ably client
+const client = new Realtime.Promise({ key: '<API-key>', clientId: '<client-ID>' });
+
+// Initialize Spaces with an Ably client
+const spaces = new Spaces(client);
+
+// Create a new space
+const space = await spaces.get('example');
+
+// Enter a space to become a member
+space.enter({ name: 'Helmut' });
+
+// Listen to all changes to all members within a space
+space.cursors.subscribe('update', (cursorUpdate) => {
+  const member = space.members.getAll().find((member) => member.connectionId === cursorUpdate.connectionId);
+  renderCursor(cursorUpdate, member);
+});
+
+// Publish cursor events to other members
+window.addEventListener('mousemove', ({ clientX, clientY }) => {
+  space.cursors.set({ position: { x: clientX, y: clientY } });
```

Comment:
// The Spaces SDK automatically batches position updates
// so you don't end up publishing millions of messages unnecessarily